x-pack/plugins/ml/public/application/jobs/jobs_list/components/datafeed_modal/datafeed_modal.tsx

```diff
@@ -0,0 +1,281 @@
+        defaultMessage: 'Error fetching data',
+      });
+      displayErrorToast(error, title);
+    }
+    setIsLoadingChartData(false);
+  }, [endDate, interval]);
+
+  useEffect(
+    function loadChartData() {
+      setIsLoadingChartData(true);
+      getChartData();
+    },
+    [endDate, interval]
+  );
+
+  return (
+    <EuiModal
+      onClose={onClose.bind(null, false)}
+      className="mlDatafeedModal"
+      style={{ height: '600px', minWidth: '800px' }}
+    >
+      <EuiModalHeader>
+        <EuiFlexGroup alignItems="center" justifyContent="spaceBetween" gutterSize="xl">
+          <EuiFlexItem grow={false}>
+            <FormattedMessage
+              id="xpack.ml.jobsList.datafeedModal.header"
+              defaultMessage="{jobId}"
+              values={{
+                jobId,
+              }}
+            />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiDatePicker
+              showTimeSelect
+              selected={endDate}
+              onChange={handleChange}
+              popoverPlacement="left-start"
+            />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiModalHeader>
+
+      <EuiModalBody>
+        <EuiTabs size="s">{renderTabs()}</EuiTabs>
+        <EuiSpacer size="m" />
+        {isLoadingChartData && <EuiLoadingChart size="l" />}
+        {!isLoadingChartData && selectedTabId === TAB_IDS.CHART && (
+          <EuiFlexGroup direction="column">
+            <EuiFlexItem grow={false}>
+              <EuiFlexGroup justifyContent="spaceBetween">
+                <EuiFlexItem grow={false}>
+                  <EuiSelect
+                    options={getIntervalOptions(bucketSpan)}
+                    value={interval}
+                    onChange={(e) => setInterval(e.target.value)}
+                    aria-label={i18n.translate(
+                      'xpack.ml.jobsList.datafeedModal.intervalSelection',
+                      {
+                        defaultMessage: 'Datafeed modal chart interval selection',
+                      }
+                    )}
+                  />
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  <EditQueryDelay datafeedConfig={datafeedConfig} />
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiFlexGroup gutterSize="none" alignItems="center">
+                <EuiFlexItem grow={false}>
+                  <EuiButtonEmpty
+                    color="primary"
+                    size="l"
+                    onClick={() => {
+                      handleEndDateChange(CHART_DIRECTION.BACK);
+                    }}
+                    iconType="arrowLeft"
+                  />
+                </EuiFlexItem>
+                <EuiFlexItem>
+                  <Chart size={CHART_SIZE}>
+                    <Settings showLegend showLegendExtra legendPosition={Position.Bottom} />
```

Comment:
Is it possible to configure the chart legend to not show the last hovered over value next to the series name? You get the values in the tooltip, so I don't think it adds any value being shown in the legend too. 

![image](https://user-images.githubusercontent.com/7405507/121224286-50afce00-c880-11eb-9026-c6bb6182a09e.png)